### PR TITLE
Feat: Added header to lasagna recipe

### PR DESCRIPTION
**Pull request by Jtommerberg**
I added an informative header to the lasagna recipe that will improve the looks.

### CODE REVIEW

File: recipes/lasagna.html [hunk 1]
```diff
@@ -38,6 +38,9 @@ <h5> For meat sauce:</h5>
     <p>
         <h5>To assemble the lasagna</h5>
     </p>
+    <p>
```
Comment (aaarland) [inline recipes/lasagna.html]: Don't use a \<h1> inside a \<p> tag.

**Review by aaarland — CHANGES_REQUESTED** [review]
Mostly good, just one change.

**Review by aaarland — APPROVED** [review]
Looks good now.

**Comment by Jtommerberg** [comment]
thanks!

**Pull request merged**